templates/index.html

```diff
@@ -2,17 +2,66 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+    <div class="jumbotron">
+        <h1 class="display-4">To-Do App</h1>
+        <p class="lead">Just another to-do app.</p>
+    </div>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+    <div class="row">
+        <div class="col-md-12">
+            <h2>Items</h2>
+            <ul class="list-group mb-4">
+                {% for item in items %}
+                    <li class="list-group-item d-flex justify-content-between align-items-center">{{ item.title }}
+                        {% if item.status == 'Not Started' %}
+                            <form id="complete-task" method="post" action={{ '/items/' + item.id|string }}>
+                                <span class="badge badge-danger badge-pill red"
+                                      onclick="complete_item()">{{ item.status }}</span>
+                            </form>
+                        {% elif item.status != 'Not Started' %}
+                            <span class="badge badge-success badge-pill red">{{ item.status }}</span>
+                        {% endif %}
+                    </li>
+                {% endfor %}
+            </ul>
+        </div>
+    </div>
+    <div>
+        <div class="float-right">
+            <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#modalAddItemForm">Add
+                Item
+            </button>
+        </div>
+        <div>
+            <form action="/items/delete" method="POST">
```

Comment:
(Hack) use POST instead of DELETE. Unable to find out why DELETE doesn't work here...

Reply:
Delete isn't supported for form actions, no.
* [MDN &lt;form> action](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attr-method)
* [StackOverflow Should PUT and DELETE be used in forms?](https://stackoverflow.com/a/5163070/243245)